examples/src/main/java/ai/timefold/solver/examples/nqueens/score/NQueensConstraintProvider.java

```diff
@@ -26,6 +29,9 @@ public Constraint[] defineConstraints(ConstraintFactory factory) {
     protected Constraint horizontalConflict(ConstraintFactory factory) {
         return factory.forEachUniquePair(Queen.class, equal(Queen::getRowIndex))
                 .penalize(SimpleScore.ONE)
+                .justifyWith((queen, otherQueen, simpleScore) -> new CustomHorizontalConflictJustification(queen.getId(),
+                        otherQueen.getId()))
+                .indictWith((queen, otherQueen) -> List.of(queen, otherQueen))
```

Comment:
Isn't this the default if you don't provide any indictments?

Reply:
It is the default behavior, but we could keep the operator to make it clearer for the user. What do you think?

Reply:
Since it's in the examples, I don't care much either way. The examples will go away anyway. Keep it if you prefer.